app/scripts/directives/showGroup.js

```diff
@@ -0,0 +1,55 @@
+import template from 'views/components/showGroup.html';
+
+angular.module('confRegistrationWebApp')
+  .component('showGroup', {
+    templateUrl: template,
+    bindings: {
+      modalInstance: '<',
+      resolve: '<'
+    },
+    controller: [function(){
+      var $ctrl = this;
+
+      $ctrl.$onInit = function() {
+        $ctrl.groupName = $ctrl.resolve.groupName;
+        $ctrl.registrationId = $ctrl.resolve.registrationId;
+        $ctrl.conference = $ctrl.resolve.conference;
+        $ctrl.getRegistration = $ctrl.resolve.getRegistration;
+        $ctrl.getRegistrantType = $ctrl.resolve.getRegistrantType;
+        $ctrl.editRegistrant = $ctrl.resolve.editRegistrant;
+        $ctrl.deleteRegistrant = $ctrl.resolve.deleteRegistrant;
+        $ctrl.registerUser = $ctrl.resolve.registerUser;
```

Comment:
Having to do all this is unfortunate. It's nice that this makes it explicit what variables you are pulling out of resolve though.

You could do
```js
const {
  groupName,
  registrationId,
  conference,
  getRegistration,
  getRegistrantType,
  editRegistrant,
  deleteRegistrant,
  registerUser
} = $ctrl.resolve;
```
to get them all as local variables but that doesn't help copy them onto $ctrl if you need them in the template.

I suppose you could clone everything from resolve onto $ctrl but that makes it less explicit and could lead to bugs if you don't know what's being copied.